src/cow-react/modules/limitOrders/pure/UnlockLimitOrders/index.tsx

```diff
@@ -44,9 +44,12 @@ export function UnlockLimitOrders({ handleUnlock }: { handleUnlock: () => void }
       )}
 
       <styledEl.ControlSection>
-        <ExternalLink href="https://cow-protocol.medium.com/the-cow-has-no-limits-342e7eae8794">
-          Learn more about limit orders ↗
-        </ExternalLink>
+        <span>
+          Learn more about{' '}
+          <ExternalLink href="https://medium.com/@cow-protocol/cow-swap-improves-the-limit-order-experience-with-partially-fillable-limit-orders-45f19143e87d">
+            partially fillable orders ↗
```

Comment:
![image](https://user-images.githubusercontent.com/2352112/232117202-c6558e6a-2951-4f34-b3c2-097e53aa77f0.png)


@avsavsavs  jajajaajj  👀

Reply:
This is just for placeholder :), I need to write a body or else you cant share the draft link :)